modules/identity-provider-create-CRD.adoc

```diff
@@ -24,18 +24,30 @@ Resource Definition (CRD) that describes it.
 
 .Procedure
 
-. Log in to the cluster as:
-+
-[source,bash]
-----
-$ command
-----
+Create a CRD file to describe the identity provider. A generic file displaying
+the structure is below.
 
-. Create the CRD for your identity provider:
-+
-[source,bash]
 ----
-$ command
+apiVersion: config.openshift.io/v1
```

Comment:
Do we need to duplicate a CRD here, too?

Reply:
I'm not certain I understand the question. We could remove this generic CRD, and instead have the instructions just indicate that an IDP specific CRD needs to be defined?

Reply:
That was the intent behind my question. :) After your discussion about using variables in modules, I wonder how many times we could reuse a module like this: https://github.com/openshift/openshift-docs/pull/13610/files#diff-21e6d7d3edc46a41e555b6ee7c74b494R1

Reply:
I think the file you've referenced makes this largely redundant. I'd much rather point them to the specific OAuth configuration for each IDP and them tell them to apply that configuration instead of having this generic file. I'll remove this file in the next push and point to yours.

Reply:
I've removed the generic CRD file, but there is a potential issue including the `deploying-resource.adoc` file in each of the assemblies. As of right now we can't define the Feature to any multi-word string. Thus the following is acceptable:

> IdentityProvider

But this is not:

> HTPasswd Identity Provider

At the beginning of each assembly we mention that only one user exists, and that a CR must be defined. At the end of the assembly we then demonstrate defining the CR, with an example for each IDP between the two.

Do you think this is sufficient, or should we add something similar to `deploying-resource.adoc` between the two modules?

Reply:
I think it might be sufficient. I think that https://auth-177--ocpdocs.netlify.com/openshift-enterprise/latest/authentication/identity_providers/configuring-htpasswd-identity-provider.html makes sense. I think you should merge it and follow up if you get pushback during QE or dev review.

Reply:
Sounds great. Thank you for checking!